x-pack/solutions/observability/plugins/apm/public/components/app/metrics/static_dashboard/dashboards/dashboard_catalog.ts

```diff
@@ -5,52 +5,69 @@
-  java: 'java',
-  'opentelemetry/java': 'opentelemetry_java',
-  'opentelemetry/java/opentelemetry-java-instrumentation': 'opentelemetry_java',
-  'opentelemetry/java/elastic': 'opentelemetry_java',
-  'opentelemetry/dotnet': 'opentelemetry_dotnet',
-  'opentelemetry/dotnet/opentelemetry-dotnet-instrumentation': 'opentelemetry_dotnet',
-  'opentelemetry/dotnet/elastic': 'opentelemetry_dotnet',
-};
+// The new dashboard file names should be added here
+export const existingDashboardFileNames = new Set([
+  'classic_apm-apm-nodejs',
+  'classic_apm-apm-java',
+  'classic_apm-otel_other-nodejs',
+  'classic_apm-otel_other-java',
+  'classic_apm-otel_other-dotnet',
+  'otel_native-otel_other-nodejs',
+  'otel_native-otel_other-java',
+  'otel_native-otel_other-dotnet',
+]);
 
-/**
- * The specially formatted comment in the `import` expression causes the corresponding webpack chunk to be named. This aids us in debugging chunk size issues.
- * See https://webpack.js.org/api/module-methods/#magic-comments
- */
-export async function loadDashboardFile(filename: string): Promise<any> {
+// The new dashboard files should be mapped here
+// + changed with the new ones (following the naming convention)
+// + similar mapping for edot needed
+//     - example: otel_native-edot-nodejs
+export async function loadDashboardFile(filename: string) {
   switch (filename) {
-    case 'nodejs': {
+    case 'classic_apm-apm-nodejs': {
       return import(
-        /* webpackChunkName: "lazyNodeJsDashboard" */
+        /* webpackChunkName: "lazyNodeJsClassicApmDashboard" */
         './nodejs.json'
       );
     }
-    case 'opentelemetry_nodejs': {
+    case 'classic_apm-otel_other-nodejs': {
       return import(
-        /* webpackChunkName: "lazyNodeJsDashboard" */
+        /* webpackChunkName: "lazyNodeJsApmOtelDashboard" */
         './opentelemetry_nodejs.json'
       );
     }
-    case 'java': {
+    case 'otel_native-otel_other-nodejs': {
       return import(
-        /* webpackChunkName: "lazyJavaDashboard" */
+        /* webpackChunkName: "lazyNodeJsOtelNativeDashboard" */
+        './opentelemetry_nodejs.json'
+      );
+    }
+    case 'classic_apm-apm-java': {
+      return import(
+        /* webpackChunkName: "lazyJavaClassicApmDashboard" */
         './java.json'
       );
     }
-    case 'opentelemetry_java': {
+    case 'classic_apm-otel_other-java': {
       return import(
-        /* webpackChunkName: "lazyJavaDashboard" */
+        /* webpackChunkName: "lazyJavaApmOtelDashboard" */
         './opentelemetry_java.json'
       );
     }
-    case 'opentelemetry_dotnet': {
+    case 'otel_native-otel_other-java': {
+      return import(
+        /* webpackChunkName: "lazyJavaOtelNativeDashboard" */
+        './opentelemetry_java.json'
+      );
+    }
+    case 'otel_native-otel_other-dotnet': {
+      return import(
+        /* webpackChunkName: "lazyDotnetOtelNativeDashboard" */
+        './opentelemetry_dotnet.json'
```

Comment:
`otel_native-otel_other-dotnet` and `classic_apm-otel_other-dotnet` apparently return the same file. Could we combine these `case`'s? Do we need to create a chunk for each? Same thing happens with other `filename`'s

Reply:
That's just because I tried to adapt the dashboard names we have to match the new naming convention - the agent team will work on adding the dashboards for each scenario and those won't be the same (but unfortunately without the dashboards I am showing what we currently have while we are still waiting for the new dashboards). 
That's the main idea of the PR: to adapt the current solution to support more dashboards for different scenarios (that's why we have those dynamic names) so it will be easier for the agent team to integrate them.